src/utils/filing_utils.py

```diff
@@ -0,0 +1,65 @@
+import argparse
+
+def get_user_inputs()->str:
+    """
+    Get user inputs for the VAS experiment type and other parameters.
+    Returns:
+        str: The name of the file containing all data.
+        str: The subject ID.
+        str: The trial type.
+        str: The trial condition.
+        str: The trial description. Usually the date in this format: MMDDYY (i.e. 01312025, which is Jan 31, 2025)
+        bool: Whether to use backup data or not.
+    
+    """
+    
+    # set up argparser
+    parser = argparse.ArgumentParser(description="VAS experimental type (10X/trial type/trial cond/desc/usebackup):")
+    parser.add_argument("--sub", type=int, default=101, help="Subject ID (default: 101)", required=True)
```

Comment:
I think for the argparser it would be be better to have the default subject name as "Test" or something which is not an actual subject, so we don't accidentally rewrite an actual data file, or make this a required argument.